package.json

```diff
@@ -95,5 +95,39 @@
   },
   "prisma": {
     "seed": "ts-node --require tsconfig-paths/register prisma/seed.ts"
+  },
+  "nx": {
+    "targets": {
```

Comment:
@vsavkin If I'm correct, this can be in the `nx.json`/`project.json` file as well?

Reply:
Agreed. If we're going to have the other config files then let's keep all config in those files.